liquidcore/home/management/commands/createusers.py

```diff
@@ -0,0 +1,41 @@
+import json
+from pathlib import Path
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+
+
+class Command(BaseCommand):
+
+    help = "Create users according to json file"
+
+    def add_arguments(self, parser):
+        parser.add_argument('users_file')
+
+    def handle(self, users_file, **options):
+        users_path = Path(users_file)
+
+        if not users_path.is_file():
+            return
+
+        with users_path.open(encoding='utf8') as f:
+            users = json.load(f)
+
+        for data in users:
+            user = User.objects.filter(username=data['username']).first()
+            if user is None:
+                print("Creating user", data['username'])
+                user = User.objects.create_user(
+                    username=data['username'],
+                    password=data['password'],
+                )
+
+            else:
+                print("Updating user", data['username'])
+
+            is_admin = bool(data.get('is_admin'))
+            user.is_superuser = is_admin
+            user.is_staff = is_admin
+            user.save()
+
+        print("Deleting users.json")
```

Comment:
A management command that says it "creates users according to json file" that silently fails and deletes said json file on success is really confusing.

Please move the `when does this run` logic out of the management command